packages/manager/modules/pci/src/components/project/storages/databases/plan.class.js

```diff
@@ -0,0 +1,98 @@
+import find from 'lodash/find';
+import groupBy from 'lodash/groupBy';
+import maxBy from 'lodash/maxBy';
+import minBy from 'lodash/minBy';
+import reduce from 'lodash/reduce';
+import some from 'lodash/some';
+
+import Region from './region.class';
+
+export default class Plan {
+  constructor({ name, description }, availability, flavors) {
```

Comment:
Can't we just destructure or properties come from different elements ?

Reply:
These properties come from different elements... The properties directly related to the plan has been destructured